charts/backstage/values.yaml

```diff
@@ -81,26 +81,38 @@ upstream:
         cpu: 1000m
         memory: 2.5Gi
         ephemeral-storage: 5Gi
+    startupProbe:
+      # This gives enough time upon container startup before the liveness and readiness probes are triggered.
+      # Giving (120s = initialDelaySeconds + failureThreshold * periodSeconds) to account for the worst case scenario.
+      httpGet:
+        path: /.backstage/health/v1/liveness
+        port: backend
+        scheme: HTTP
+      initialDelaySeconds: 30
+      timeoutSeconds: 4
+      periodSeconds: 20
+      successThreshold: 1
+      failureThreshold: 3
     readinessProbe:
       failureThreshold: 3
       httpGet:
-        path: /healthcheck
-        port: 7007
+        path: /.backstage/health/v1/readiness
+        port: backend
         scheme: HTTP
-      initialDelaySeconds: 30
+      # initialDelaySeconds: 30
       periodSeconds: 10
       successThreshold: 2
-      timeoutSeconds: 2
+      timeoutSeconds: 4
     livenessProbe:
       failureThreshold: 3
       httpGet:
-        path: /healthcheck
-        port: 7007
+        path: /.backstage/health/v1/liveness
+        port: backend
         scheme: HTTP
-      initialDelaySeconds: 60
+      # initialDelaySeconds: 60
```

Comment:
why commented out? a comment explaining why would be good for future-us to see :D

Reply:
We ended up finding out some more information about how each of the probes are triggered. Omitting the delay for the liveness and readiness probes will mean that they will trigger after a successful startup probe. This should help to reduce some of the start up time. https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#configure-probes

Reply:
Agreed that maybe a comment explaining would be helpful in the code though as I didn't know about this until last night.

Reply:
Comment added in [`a226884` (#50)](https://github.com/redhat-developer/rhdh-chart/pull/50/commits/a226884a9a3c872383c3b3797d1151f3a356cfa5)